ui/app/routes/variables/new.js

```diff
@@ -0,0 +1,17 @@
+import Route from '@ember/routing/route';
+
+export default class VariablesNewRoute extends Route {
+  model() {
+    return this.store.findAll('namespace').then(() => {
+      return this.store.createRecord('variable');
```

Comment:
`question`: Do we need to `.then` our `createRecord`? We're not reliant on whether the first promise resolves or rejects according to this logic?

It appears that we want to have all namespaces available when the route loads and then create a new variable upon entering this route.

Also, what happens if either promise returns a non `2XX` code?

Maybe this should be:

```
try {
 // findAll namespaces
 // createRecord
} catch (e) {
  // error logic
}
```

Reply:
Good call, and I think this is possibly not needed. This was taking a page from the `jobs/run` playbook, but by definition we'll already have namespaces on this route. Simplifying now.